packages/shorebird_cli/lib/src/archive_analysis/ios_archive_differ.dart

```diff
@@ -32,15 +33,18 @@ class IosArchiveDiffer extends ArchiveDiffer {
   /// archives at the two provided paths. This method will also unisgn mach-o
   /// binaries in the archives before computing the diff.
   @override
-  FileSetDiff changedFiles(String oldArchivePath, String newArchivePath) {
-    final oldPathHashes = fileHashes(File(oldArchivePath));
-    final newPathHashes = fileHashes(File(newArchivePath));
-
-    _updateHashes(
+  Future<FileSetDiff> changedFiles(
+    String oldArchivePath,
+    String newArchivePath,
+  ) async {
+    var oldPathHashes = await fileHashes(File(oldArchivePath));
+    var newPathHashes = await fileHashes(File(newArchivePath));
+
+    oldPathHashes = await _updateHashes(
       archivePath: oldArchivePath,
       pathHashes: oldPathHashes,
     );
-    _updateHashes(
+    newPathHashes = await _updateHashes(
       archivePath: newArchivePath,
       pathHashes: newPathHashes,
     );
```

Comment:
Can we do these in parallel or do they need to be done in series?

Reply:
They could be done in parallel, will update

Reply:
Will do in a future PR so to get this in the current release.